app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/General.php

```diff
@@ -41,20 +43,28 @@ class General extends AbstractModifier
      */
     private $attributeRepository;
 
+    /**
+     * @var StockStateInterface
+     */
+    private $stockState;
+
     /**
      * @param LocatorInterface $locator
      * @param ArrayManager $arrayManager
+     * @param StockStateInterface $stockState
      * @param AttributeRepositoryInterface|null $attributeRepository
      */
     public function __construct(
         LocatorInterface $locator,
         ArrayManager $arrayManager,
+        StockStateInterface $stockState,
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)